src/controllers/app.controller.tasks.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Post, Param, Query, Body, Patch, Delete } from '@nestjs/common';
+import { TasksPrismaRepository } from 'src/repositories/TasksPrismaRepository';
+import { Task } from 'src/entities/Task';
+import { CreateTaskDTO } from 'src/dtos/entities.Task.createTaskDTO';
+import { UpdateTaskDTO } from 'src/dtos/entities.Task.updateTaskDTO';
+import { randomUUID } from 'crypto';
+
+
+@Controller('tasks')
+export class AppControllerTasks {
+  constructor(
+    private readonly prisma: TasksPrismaRepository
+  ) { }
+
+  //GET AREA
+
+  @Get()
+  async getAllTasks() {
+    return this.prisma.findAll();
+  }
+
+  @Get('/day/')
```

Comment:
```suggestion
  @Get('day/')
```
O framework já faz a adição dessa barra extra, n precisa especificar